src/components/auth/registerUser.js

```diff
@@ -0,0 +1,228 @@
+import React, { Component } from "react";
+import Input from "../common/input";
+import Button from "../common/button";
+import "./../../assets/authAssets/test.css";
+import "bootstrap/dist/css/bootstrap.min.css";
```

Comment:
Remove all the bootstrap cdns since we wont be using them

Reply:
Removed the bootstrap